lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -465,6 +466,27 @@ public function select($select = null) : self
         return $this->add('select', $selects);
     }
 
+    /**
+     * Adds a DISTINCT flag to this query.
+     *
+     * <code>
+     *     $qb = $conn->createQueryBuilder()
+     *         ->select('u.id')
+     *         ->distinct()
+     *         ->from('users', 'u');
+     * </code>
+     *
+     * @param bool $flag If true, then DISTINCT is added to the query
```

Comment:
In what scenario is that parameter going to be useful?

Reply:
Do you mean, that there is no sense to mention this parameter in the docstring, since it is self evident? And by the way, as for `develop` branch, should I make a new merge request to the master branch to fix this?

Reply:
No I mean, maybe we should remove the parameter entirely?

> And by the way, as for develop branch, should I make a new merge request to the master branch to fix this?

You can edit your PR to target master, and rebase your branch on it: `git rebase --onto master origin/develop develop` (you should probably not have named your local branch `develop` BTW, it makes this a bit confusing :P)

Reply:
Ups, I've never done that magic yet and made another fork for another PR. So, I think I will close this one and start another PR, if you don't mind. And I will take into account all you suggestions, probably except `final` key word